virttest/qemu_devices/qcontainer.py

```diff
@@ -1509,6 +1509,9 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus,
                 # Please refer to qemu commit d657c0c.
                 cache = cache not in ['none', 'directsync'] and 'none' or cache
             devices[-1].set_param('cache', cache)
```

Comment:
Be better to set cache=None and limit  devices[-1].set_param('cache', cache) setting, like
  if cache:
      devices[-1].set_param('cache', cache)